test/internal/cmd/generate-gateway-api-urls/main.go

```diff
@@ -12,26 +12,29 @@ import (
 	"text/template"
 )
 
-//go:generate go run . -crds-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
+//go:generate go run . -standard-crds-url $STANDARD_CRDS_URL -crds-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
```

Comment:
```suggestion
//go:generate go run . -crds-standard-url $STANDARD_CRDS_URL -crds-experimental-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
```

Reply:
If this change is taken, the names of env var should also be changed. So WDYT of the env vars here?

Reply:
Yes, good point. Let's change them. I feel this will be more natural (and easier to group by in head) to have a `CRDS_` prefix

Reply:
updated. Also included changes in Makefile.